--- ../src-base/minecraft/net/minecraft/block/BlockButton.java
+++ ../src-work/minecraft/net/minecraft/block/BlockButton.java
@@ -95,7 +95,7 @@
         IBlockState iblockstate = p_181088_0_.func_180495_p(blockpos);
         boolean flag = iblockstate.func_193401_d(p_181088_0_, blockpos, p_181088_2_) == BlockFaceShape.SOLID;
         Block block = iblockstate.func_177230_c();
-        return p_181088_2_ == EnumFacing.UP ? block == Blocks.field_150438_bZ || !func_193384_b(block) && flag : !func_193382_c(block) && flag;
+        return p_181088_2_ == EnumFacing.UP ? iblockstate.func_185896_q() || !func_193384_b(block) && flag : !func_193382_c(block) && flag;
     }
 
     public IBlockState func_180642_a(World p_180642_1_, BlockPos p_180642_2_, EnumFacing p_180642_3_, float p_180642_4_, float p_180642_5_, float p_180642_6_, int p_180642_7_, EntityLivingBase p_180642_8_)
