diff -U minecraft\net\minecraft\entity\player\EntityPlayer.java minecraft_patched\net\minecraft\entity\player\EntityPlayer.java
--- minecraft\net\minecraft\entity\player\EntityPlayer.java
+++ minecraft_patched\net\minecraft\entity\player\EntityPlayer.java
@@ -337,12 +337,12 @@
         }
 
         int i = 29999999;
-        double d4 = MathHelper.func_151237_a(this.field_70165_t, -2.9999999E7D, 2.9999999E7D);
-        double d5 = MathHelper.func_151237_a(this.field_70161_v, -2.9999999E7D, 2.9999999E7D);
+        double d3 = MathHelper.func_151237_a(this.field_70165_t, -2.9999999E7D, 2.9999999E7D);
+        double d4 = MathHelper.func_151237_a(this.field_70161_v, -2.9999999E7D, 2.9999999E7D);
 
-        if (d4 != this.field_70165_t || d5 != this.field_70161_v)
+        if (d3 != this.field_70165_t || d4 != this.field_70161_v)
         {
-            this.func_70107_b(d4, this.field_70163_u, d5);
+            this.func_70107_b(d3, this.field_70163_u, d4);
         }
     }
 
