diff -U minecraft\net\minecraft\entity\EntityLiving.java minecraft_patched\net\minecraft\entity\EntityLiving.java
--- minecraft\net\minecraft\entity\EntityLiving.java
+++ minecraft_patched\net\minecraft\entity\EntityLiving.java
@@ -1250,6 +1250,6 @@
         private SpawnPlacementType(String p_i45893_1_, int p_i45893_2_) {}
 
         @SideOnly(Side.SERVER)
-
+		private SpawnPlacementType(String p_i45893_1_, int p_i45893_2_) {}
     }
 }
