--- ../src-base/minecraft/net/minecraft/entity/passive/EntityTameable.java
+++ ../src-work/minecraft/net/minecraft/entity/passive/EntityTameable.java
@@ -38,7 +38,7 @@
     {
         super.func_70088_a();
         this.field_70180_af.func_187214_a(field_184755_bv, Byte.valueOf((byte)0));
-        this.field_70180_af.func_187214_a(field_184756_bw, Optional.<T>absent());
+        this.field_70180_af.func_187214_a(field_184756_bw, Optional.absent());
     }
 
     public void func_70014_b(NBTTagCompound p_70014_1_)
@@ -185,7 +185,7 @@
 
     public void func_184754_b(@Nullable UUID p_184754_1_)
     {
-        this.field_70180_af.func_187227_b(field_184756_bw, Optional.<T>fromNullable(p_184754_1_));
+        this.field_70180_af.func_187227_b(field_184756_bw, Optional.fromNullable(p_184754_1_));
     }
 
     public void func_193101_c(EntityPlayer p_193101_1_)
