--- ../src-base/minecraft/net/minecraft/entity/passive/EntityLlama.java
+++ ../src-work/minecraft/net/minecraft/entity/passive/EntityLlama.java
@@ -518,7 +518,7 @@
         {
             public AIDefendTarget(EntityLlama p_i47285_1_)
             {
-                super(p_i47285_1_, EntityWolf.class, 16, false, true, (Predicate <? super T >)null);
+                super(p_i47285_1_, EntityWolf.class, 16, false, true, null);
             }
 
             public boolean func_75250_a()
