--- ../src-base/minecraft/net/minecraft/client/resources/model/ModelRotation.java
+++ ../src-work/minecraft/net/minecraft/client/resources/model/ModelRotation.java
@@ -2,6 +2,8 @@
 
 import com.google.common.base.Optional;
 import com.google.common.collect.Maps;
+
+import java.nio.FloatBuffer;
 import java.util.Map;
 import net.minecraft.src.Reflector;
 import net.minecraft.util.EnumFacing;
@@ -47,18 +49,18 @@
         return p_177521_0_ * 360 + p_177521_1_;
     }
 
-    private ModelRotation(String p_i77_3_, int p_i77_4_, int p_i77_5_, int p_i77_6_)
+    private ModelRotation(String p_i46087_1_, int p_i46087_2_, int p_i46087_3_, int p_i46087_4_)
     {
-        this.field_177545_r = func_177521_b(p_i77_5_, p_i77_6_);
+        this.field_177545_r = func_177521_b(p_i46087_3_, p_i46087_4_);
         this.field_177544_s = new Matrix4f();
         Matrix4f matrix4f = new Matrix4f();
         matrix4f.setIdentity();
-        Matrix4f.rotate((float)(-p_i77_5_) * 0.017453292F, new Vector3f(1.0F, 0.0F, 0.0F), matrix4f, matrix4f);
-        this.field_177543_t = MathHelper.func_76130_a(p_i77_5_ / 90);
+        Matrix4f.rotate((float)(-p_i46087_3_) * 0.017453292F, new Vector3f(1.0F, 0.0F, 0.0F), matrix4f, matrix4f);
+        this.field_177543_t = MathHelper.func_76130_a(p_i46087_3_ / 90);
         Matrix4f matrix4f1 = new Matrix4f();
         matrix4f1.setIdentity();
-        Matrix4f.rotate((float)(-p_i77_6_) * 0.017453292F, new Vector3f(0.0F, 1.0F, 0.0F), matrix4f1, matrix4f1);
-        this.field_177542_u = MathHelper.func_76130_a(p_i77_6_ / 90);
+        Matrix4f.rotate((float)(-p_i46087_4_) * 0.017453292F, new Vector3f(0.0F, 1.0F, 0.0F), matrix4f1, matrix4f1);
+        this.field_177542_u = MathHelper.func_76130_a(p_i46087_4_ / 90);
         Matrix4f.mul(matrix4f1, matrix4f, this.field_177544_s);
     }
 
@@ -123,7 +125,16 @@
 
     public javax.vecmath.Matrix4f getMatrix()
     {
-        return Reflector.ForgeHooksClient_getMatrix.exists() ? (javax.vecmath.Matrix4f)Reflector.call(Reflector.ForgeHooksClient_getMatrix, new Object[] {this}): new javax.vecmath.Matrix4f(this.func_177525_a());
+    	if(Reflector.ForgeHooksClient_getMatrix.exists())
+    	{
+    		return (javax.vecmath.Matrix4f)Reflector.call(Reflector.ForgeHooksClient_getMatrix, new Object[] {this});
+    	}
+    	else
+    	{
+    		FloatBuffer buffer = FloatBuffer.allocate(16);
+    		this.func_177525_a().store(buffer);
+    		return new javax.vecmath.Matrix4f(buffer.array());
+    	}
     }
 
     public EnumFacing rotate(EnumFacing p_rotate_1_)
