--- ../src-base/minecraft/net/minecraft/block/state/BlockStateContainer.java
+++ ../src-work/minecraft/net/minecraft/block/state/BlockStateContainer.java
@@ -102,7 +102,7 @@
             blockstatecontainer$stateimplementation1.func_177235_a(map2);
         }
 
-        this.field_177625_e = ImmutableList.copyOf(list);
+        this.field_177625_e = (ImmutableList) ImmutableList.copyOf(list);
     }
 
     public static <T extends Comparable<T>> String func_185919_a(Block p_185919_0_, IProperty<T> p_185919_1_)
@@ -143,7 +143,7 @@
         while (unmodifiableiterator.hasNext())
         {
             IProperty<?> iproperty = (IProperty)unmodifiableiterator.next();
-            list.add(iproperty.func_177700_c());
+            list.add((Iterable) iproperty.func_177700_c());
         }
 
         return list;
