--- ../src-base/minecraft/net/minecraft/util/EnumFacing.java
+++ ../src-work/minecraft/net/minecraft/util/EnumFacing.java
@@ -31,15 +31,15 @@
     private static final EnumFacing[] $VALUES = new EnumFacing[]{DOWN, UP, NORTH, SOUTH, WEST, EAST};
 
     @SideOnly(Side.CLIENT)
-    private EnumFacing(String p_i58_3_, int p_i58_4_, int p_i58_5_, int p_i58_6_, int p_i58_7_, String p_i58_8_, EnumFacing.AxisDirection p_i58_9_, EnumFacing.Axis p_i58_10_, Vec3i p_i58_11_)
+    private EnumFacing(String p_i46016_1_, int p_i46016_2_, int p_i46016_3_, int p_i46016_4_, int p_i46016_5_, String p_i46016_6_, EnumFacing.AxisDirection p_i46016_7_, EnumFacing.Axis p_i46016_8_, Vec3i p_i46016_9_)
     {
-        this.field_176748_g = p_i58_5_;
-        this.field_176760_i = p_i58_7_;
-        this.field_176759_h = p_i58_6_;
-        this.field_176757_j = p_i58_8_;
-        this.field_176758_k = p_i58_10_;
-        this.field_176755_l = p_i58_9_;
-        this.field_176756_m = p_i58_11_;
+        this.field_176748_g = p_i46016_3_;
+        this.field_176760_i = p_i46016_5_;
+        this.field_176759_h = p_i46016_4_;
+        this.field_176757_j = p_i46016_6_;
+        this.field_176758_k = p_i46016_8_;
+        this.field_176755_l = p_i46016_7_;
+        this.field_176756_m = p_i46016_9_;
     }
 
     @SideOnly(Side.SERVER)
@@ -420,10 +420,10 @@
         private static final EnumFacing.Axis[] $VALUES = new EnumFacing.Axis[]{X, Y, Z};
 
         @SideOnly(Side.CLIENT)
-        private Axis(String p_i73_3_, int p_i73_4_, String p_i73_5_, EnumFacing.Plane p_i73_6_)
+        private Axis(String p_i46015_1_, int p_i46015_2_, String p_i46015_3_, EnumFacing.Plane p_i46015_4_)
         {
-            this.field_176726_e = p_i73_5_;
-            this.field_176723_f = p_i73_6_;
+            this.field_176726_e = p_i46015_3_;
+            this.field_176723_f = p_i46015_4_;
         }
 
         @SideOnly(Side.SERVER)
@@ -497,10 +497,10 @@
         private static final EnumFacing.AxisDirection[] $VALUES = new EnumFacing.AxisDirection[]{POSITIVE, NEGATIVE};
 
         @SideOnly(Side.CLIENT)
-        private AxisDirection(String p_i71_3_, int p_i71_4_, int p_i71_5_, String p_i71_6_)
+        private AxisDirection(String p_i46014_1_, int p_i46014_2_, int p_i46014_3_, String p_i46014_4_)
         {
-            this.field_179528_c = p_i71_5_;
-            this.field_179525_d = p_i71_6_;
+            this.field_179528_c = p_i46014_3_;
+            this.field_179525_d = p_i46014_4_;
         }
 
         @SideOnly(Side.SERVER)
@@ -521,14 +521,14 @@
         }
     }
 
-    public static enum Plane implements Predicate, Iterable {
+    public static enum Plane implements Predicate, Iterable<EnumFacing> {
         HORIZONTAL("HORIZONTAL", 0),
         VERTICAL("VERTICAL", 1);
 
         private static final EnumFacing.Plane[] $VALUES = new EnumFacing.Plane[]{HORIZONTAL, VERTICAL};
 
         @SideOnly(Side.CLIENT)
-        private Plane(String p_i72_3_, int p_i72_4_)
+        private Plane(String p_i46013_1_, int p_i46013_2_)
         {
         }
 
