--- ../src-base/minecraft/net/minecraft/client/renderer/entity/RenderZombie.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/entity/RenderZombie.java
@@ -22,8 +22,8 @@
     private static final ResourceLocation field_110864_q = new ResourceLocation("textures/entity/zombie/zombie_villager.png");
     private final ModelBiped field_82434_o;
     private final ModelZombieVillager field_82432_p;
-    private final List<LayerRenderer<EntityZombie>> field_177121_n;
-    private final List<LayerRenderer<EntityZombie>> field_177122_o;
+    private final List<LayerRenderer> field_177121_n;
+    private final List<LayerRenderer> field_177122_o;
 
     public RenderZombie(RenderManager p_i46127_1_)
     {
