--- ../src-base/minecraft/net/minecraft/client/renderer/entity/RendererLivingEntity.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/entity/RendererLivingEntity.java
@@ -33,13 +33,13 @@
 import shadersmod.client.Shaders;
 
 @SideOnly(Side.CLIENT)
-public abstract class RendererLivingEntity extends Render
+public abstract class RendererLivingEntity<T extends EntityLivingBase> extends Render<T>
 {
     private static final Logger field_147923_a = LogManager.getLogger();
     private static final DynamicTexture field_177096_e = new DynamicTexture(16, 16);
     protected ModelBase field_77045_g;
     protected FloatBuffer field_177095_g = GLAllocation.func_74529_h(4);
-    protected List field_177097_h = Lists.newArrayList();
+    protected List<LayerRenderer> field_177097_h = Lists.newArrayList();
     protected boolean field_177098_i = false;
     public static float NAME_TAG_RANGE = 64.0F;
     public static float NAME_TAG_RANGE_SNEAK = 32.0F;
@@ -485,8 +485,8 @@
     protected void func_77041_b(EntityLivingBase p_77041_1_, float p_77041_2_)
     {
     }
-
-    public void func_177067_a(EntityLivingBase p_177067_1_, double p_177067_2_, double p_177067_4_, double p_177067_6_)
+    
+    public void func_177067_a(T p_177067_1_, double p_177067_2_, double p_177067_4_, double p_177067_6_)
     {
         if (!Reflector.RenderLivingEvent_Specials_Pre_Constructor.exists() || !Reflector.postForgeBusEvent(Reflector.RenderLivingEvent_Specials_Pre_Constructor, new Object[] {p_177067_1_, this, Double.valueOf(p_177067_2_), Double.valueOf(p_177067_4_), Double.valueOf(p_177067_6_)}))
         {
