--- ../src-base/minecraft/net/minecraft/client/entity/AbstractClientPlayer.java
+++ ../src-work/minecraft/net/minecraft/client/entity/AbstractClientPlayer.java
@@ -189,9 +189,9 @@
             IImageBuffer iimagebuffer = new IImageBuffer()
             {
                 ImageBufferDownload ibd = new ImageBufferDownload();
-                public BufferedImage func_78432_a(BufferedImage p_78432_1_)
+                public BufferedImage func_78432_a(BufferedImage image)
                 {
-                    return AbstractClientPlayer.this.parseCape(p_78432_1_);
+                    return AbstractClientPlayer.this.parseCape(image);
                 }
                 public void func_152634_a()
                 {
