--- ../src-base/minecraft/net/minecraft/advancements/FunctionManager.java
+++ ../src-work/minecraft/net/minecraft/advancements/FunctionManager.java
@@ -116,9 +116,9 @@
                 int j = 0;
                 FunctionObject.Entry[] afunctionobject$entry = p_194019_1_.func_193528_a();
 
-                for (lvt_6_1_ = afunctionobject$entry.length - 1; k >= 0; --k)
+                for (int l = afunctionobject$entry.length - 1; l >= 0; --l)
                 {
-                    this.field_194020_g.push(new FunctionManager.QueuedCommand(this, p_194019_2_, afunctionobject$entry[k]));
+                    this.field_194020_g.push(new FunctionManager.QueuedCommand(this, p_194019_2_, afunctionobject$entry[l]));
                 }
 
                 while (true)
