--- ../src-base/minecraft/net/minecraft/client/resources/AbstractResourcePack.java
+++ ../src-work/minecraft/net/minecraft/client/resources/AbstractResourcePack.java
@@ -25,7 +25,7 @@
 public abstract class AbstractResourcePack implements IResourcePack
 {
     private static final Logger field_110598_a = LogManager.getLogger();
-    protected final File field_110597_b;
+    public final File field_110597_b;
 
     public AbstractResourcePack(File p_i1287_1_)
     {
