--- ../src-base/minecraft/net/minecraft/client/renderer/BlockModelRenderer.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/BlockModelRenderer.java
@@ -182,7 +182,7 @@
         return flag;
     }
 
-    private void renderModelAmbientOcclusionQuads(IBlockAccess p_renderModelAmbientOcclusionQuads_1_, Block p_renderModelAmbientOcclusionQuads_2_, BlockPos p_renderModelAmbientOcclusionQuads_3_, WorldRenderer p_renderModelAmbientOcclusionQuads_4_, List p_renderModelAmbientOcclusionQuads_5_, RenderEnv p_renderModelAmbientOcclusionQuads_6_)
+    private void renderModelAmbientOcclusionQuads(IBlockAccess p_renderModelAmbientOcclusionQuads_1_, Block p_renderModelAmbientOcclusionQuads_2_, BlockPos p_renderModelAmbientOcclusionQuads_3_, WorldRenderer p_renderModelAmbientOcclusionQuads_4_, List<BakedQuad> p_renderModelAmbientOcclusionQuads_5_, RenderEnv p_renderModelAmbientOcclusionQuads_6_)
     {
         float[] afloat = p_renderModelAmbientOcclusionQuads_6_.getQuadBounds();
         BitSet bitset = p_renderModelAmbientOcclusionQuads_6_.getBoundsFlags();
@@ -346,7 +346,7 @@
         }
     }
 
-    private void renderModelStandardQuads(IBlockAccess p_renderModelStandardQuads_1_, Block p_renderModelStandardQuads_2_, BlockPos p_renderModelStandardQuads_3_, EnumFacing p_renderModelStandardQuads_4_, int p_renderModelStandardQuads_5_, boolean p_renderModelStandardQuads_6_, WorldRenderer p_renderModelStandardQuads_7_, List p_renderModelStandardQuads_8_, RenderEnv p_renderModelStandardQuads_9_)
+    private void renderModelStandardQuads(IBlockAccess p_renderModelStandardQuads_1_, Block p_renderModelStandardQuads_2_, BlockPos p_renderModelStandardQuads_3_, EnumFacing p_renderModelStandardQuads_4_, int p_renderModelStandardQuads_5_, boolean p_renderModelStandardQuads_6_, WorldRenderer p_renderModelStandardQuads_7_, List<BakedQuad> p_renderModelStandardQuads_8_, RenderEnv p_renderModelStandardQuads_9_)
     {
         BitSet bitset = p_renderModelStandardQuads_9_.getBoundsFlags();
         IBlockState iblockstate = p_renderModelStandardQuads_9_.getBlockState();
@@ -471,7 +471,7 @@
         this.func_178262_a(p_178266_1_, p_178266_3_, f, f1, f2);
     }
 
-    private void func_178264_a(float p_178264_1_, float p_178264_2_, float p_178264_3_, float p_178264_4_, List p_178264_5_)
+    private void func_178264_a(float p_178264_1_, float p_178264_2_, float p_178264_3_, float p_178264_4_, List<BakedQuad> p_178264_5_)
     {
         Tessellator tessellator = Tessellator.func_178181_a();
         WorldRenderer worldrenderer = tessellator.func_178180_c();
@@ -769,15 +769,15 @@
         private static final BlockModelRenderer.EnumNeighborInfo[] field_178282_n = new BlockModelRenderer.EnumNeighborInfo[6];
         private static final BlockModelRenderer.EnumNeighborInfo[] $VALUES = new BlockModelRenderer.EnumNeighborInfo[]{DOWN, UP, NORTH, SOUTH, WEST, EAST};
 
-        private EnumNeighborInfo(String p_i0_3_, int p_i0_4_, EnumFacing[] p_i0_5_, float p_i0_6_, boolean p_i0_7_, BlockModelRenderer.Orientation[] p_i0_8_, BlockModelRenderer.Orientation[] p_i0_9_, BlockModelRenderer.Orientation[] p_i0_10_, BlockModelRenderer.Orientation[] p_i0_11_)
+        private EnumNeighborInfo(String p_i46236_1_, int p_i46236_2_, EnumFacing[] p_i46236_3_, float p_i46236_4_, boolean p_i46236_5_, BlockModelRenderer.Orientation[] p_i46236_6_, BlockModelRenderer.Orientation[] p_i46236_7_, BlockModelRenderer.Orientation[] p_i46236_8_, BlockModelRenderer.Orientation[] p_i46236_9_)
         {
-            this.field_178276_g = p_i0_5_;
-            this.field_178288_h = p_i0_6_;
-            this.field_178289_i = p_i0_7_;
-            this.field_178286_j = p_i0_8_;
-            this.field_178287_k = p_i0_9_;
-            this.field_178284_l = p_i0_10_;
-            this.field_178285_m = p_i0_11_;
+            this.field_178276_g = p_i46236_3_;
+            this.field_178288_h = p_i46236_4_;
+            this.field_178289_i = p_i46236_5_;
+            this.field_178286_j = p_i46236_6_;
+            this.field_178287_k = p_i46236_7_;
+            this.field_178284_l = p_i46236_8_;
+            this.field_178285_m = p_i46236_9_;
         }
 
         public static BlockModelRenderer.EnumNeighborInfo func_178273_a(EnumFacing p_178273_0_)
@@ -815,9 +815,9 @@
         protected final int field_178229_m;
         private static final BlockModelRenderer.Orientation[] $VALUES = new BlockModelRenderer.Orientation[]{DOWN, UP, NORTH, SOUTH, WEST, EAST, FLIP_DOWN, FLIP_UP, FLIP_NORTH, FLIP_SOUTH, FLIP_WEST, FLIP_EAST};
 
-        private Orientation(String p_i1_3_, int p_i1_4_, EnumFacing p_i1_5_, boolean p_i1_6_)
+        private Orientation(String p_i46233_1_, int p_i46233_2_, EnumFacing p_i46233_3_, boolean p_i46233_4_)
         {
-            this.field_178229_m = p_i1_5_.func_176745_a() + (p_i1_6_ ? EnumFacing.values().length : 0);
+            this.field_178229_m = p_i46233_3_.func_176745_a() + (p_i46233_4_ ? EnumFacing.values().length : 0);
         }
     }
 
@@ -838,12 +838,12 @@
         private static final BlockModelRenderer.VertexTranslations[] field_178199_k = new BlockModelRenderer.VertexTranslations[6];
         private static final BlockModelRenderer.VertexTranslations[] $VALUES = new BlockModelRenderer.VertexTranslations[]{DOWN, UP, NORTH, SOUTH, WEST, EAST};
 
-        private VertexTranslations(String p_i2_3_, int p_i2_4_, int p_i2_5_, int p_i2_6_, int p_i2_7_, int p_i2_8_)
+        private VertexTranslations(String p_i46234_1_, int p_i46234_2_, int p_i46234_3_, int p_i46234_4_, int p_i46234_5_, int p_i46234_6_)
         {
-            this.field_178191_g = p_i2_5_;
-            this.field_178200_h = p_i2_6_;
-            this.field_178201_i = p_i2_7_;
-            this.field_178198_j = p_i2_8_;
+            this.field_178191_g = p_i46234_3_;
+            this.field_178200_h = p_i46234_4_;
+            this.field_178201_i = p_i46234_5_;
+            this.field_178198_j = p_i46234_6_;
         }
 
         public static BlockModelRenderer.VertexTranslations func_178184_a(EnumFacing p_178184_0_)
