--- ../src-base/minecraft/net/minecraft/client/renderer/RenderGlobal.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/RenderGlobal.java
@@ -184,8 +184,8 @@
     public Set chunksToUpdateForced = new LinkedHashSet();
     private Set<RenderChunk> chunksToUpdatePrev = new ObjectLinkedOpenHashSet();
     private Deque visibilityDeque = new ArrayDeque();
-    private List renderInfosEntities = new ArrayList(1024);
-    private List renderInfosTileEntities = new ArrayList(1024);
+    private List<ContainerLocalRenderInformation> renderInfosEntities = new ArrayList(1024);
+    private List<ContainerLocalRenderInformation> renderInfosTileEntities = new ArrayList(1024);
     private List renderInfosNormal = new ArrayList(1024);
     private List renderInfosEntitiesNormal = new ArrayList(1024);
     private List renderInfosTileEntitiesNormal = new ArrayList(1024);
@@ -213,9 +213,9 @@
         this.field_72777_q = p_i1249_1_;
         this.field_175010_j = p_i1249_1_.func_175598_ae();
         this.field_72770_i = p_i1249_1_.func_110434_K();
-        this.field_72770_i.func_110577_a(field_175006_g);
-        GlStateManager.func_187421_b(3553, 10242, 10497);
-        GlStateManager.func_187421_b(3553, 10243, 10497);
+        //this.renderEngine.bindTexture(FORCEFIELD_TEXTURES);
+        //GlStateManager.glTexParameteri(3553, 10242, 10497);
+        //GlStateManager.glTexParameteri(3553, 10243, 10497);
         GlStateManager.func_179144_i(0);
         this.func_174971_n();
         this.field_175005_X = OpenGlHelper.func_176075_f();
@@ -3448,11 +3448,11 @@
             int field_178035_c;
             boolean cacheable = false;
 
-            public ContainerLocalRenderInformation(RenderChunk p_i96_1_, EnumFacing p_i96_2_, int p_i96_3_)
+            public ContainerLocalRenderInformation(RenderChunk p_i46248_2_, EnumFacing p_i46248_3_, int p_i46248_4_)
             {
-                this.field_178036_a = p_i96_1_;
-                this.field_178034_b = p_i96_2_;
-                this.field_178035_c = p_i96_3_;
+                this.field_178036_a = p_i46248_2_;
+                this.field_178034_b = p_i46248_3_;
+                this.field_178035_c = p_i46248_4_;
             }
 
             public void func_189561_a(byte p_189561_1_, EnumFacing p_189561_2_)
