--- /net/minecraft/client/resources/AbstractResourcePack.java
+++ /net/minecraft/client/resources/AbstractResourcePack.java
@@ -24,7 +24,7 @@
 public abstract class AbstractResourcePack implements IResourcePack
 {
     private static final Logger resourceLog = LogManager.getLogger();
-    protected final File resourcePackFile;
+    public final File resourcePackFile;
     private static final String __OBFID = "CL_00001072";
 
     public AbstractResourcePack(File p_i1287_1_)
