--- /net/minecraft/client/renderer/block/model/BakedQuad.java
+++ /net/minecraft/client/renderer/block/model/BakedQuad.java
@@ -6,12 +6,13 @@
 import net.minecraftforge.fml.relauncher.SideOnly;
 
 @SideOnly(Side.CLIENT)
-public class BakedQuad
+public class BakedQuad implements net.minecraftforge.client.model.pipeline.IVertexProducer
 {
     /**
      * Joined 4 vertex records, each has 7 fields (x, y, z, shadeColor, u, v, <unused>), see
      * FaceBakery.storeVertexData()
      */
+    @Override public void pipe(net.minecraftforge.client.model.pipeline.IVertexConsumer consumer) { net.minecraftforge.client.model.pipeline.LightUtil.putBakedQuad(consumer, this); }
     protected final int[] vertexData;
     protected final int tintIndex;
     protected final EnumFacing face;
