--- /net/minecraft/src/Config.java
+++ /net/minecraft/src/Config.java
@@ -1567,7 +1567,7 @@
 
     public static void drawFps()
     {
-        Minecraft minecraft = minecraft;
+        Minecraft minecraft = Config.minecraft;
         int fps = Minecraft.getDebugFPS();
         String updates = getUpdates(minecraft.debug);
         int renderersActive = minecraft.renderGlobal.getCountActiveRenderers();
