--- /net/minecraft/util/EnumFacing.java
+++ /net/minecraft/util/EnumFacing.java
@@ -484,7 +484,7 @@
         }
     }
 
-    public static enum Plane implements Predicate, Iterable {
+    public static enum Plane implements Predicate, Iterable<EnumFacing> {
         HORIZONTAL("HORIZONTAL", 0),
         VERTICAL("VERTICAL", 1);
         private static final EnumFacing.Plane[] $VALUES = new EnumFacing.Plane[]{HORIZONTAL, VERTICAL};
@@ -495,8 +495,6 @@
 
         @SideOnly(Side.CLIENT)
         private Plane(String p_i46013_1_, int p_i46013_2_) {}
-
-        @SideOnly(Side.SERVER)
 
         /**
          * All EnumFacing values for this Plane
