--- /net/minecraft/client/renderer/block/model/FaceBakery.java
+++ /net/minecraft/client/renderer/block/model/FaceBakery.java
@@ -21,11 +21,6 @@
     private static final double field_178418_a = 1.0D / Math.cos(0.39269908169872414D) - 1.0D;
     private static final double field_178417_b = 1.0D / Math.cos((Math.PI / 4D)) - 1.0D;
     private static final String __OBFID = "CL_00002490";
-
-    public BakedQuad makeBakedQuad(Vector3f posFrom, Vector3f posTo, BlockPartFace face, TextureAtlasSprite sprite, EnumFacing facing, ModelRotation modelRotationIn, BlockPartRotation partRotation, boolean uvLocked, boolean shade)
-    {
-        return this.makeBakedQuad(posFrom, posTo, face, sprite, facing, modelRotationIn, partRotation, uvLocked, shade);
-    }
 
     public BakedQuad makeBakedQuad(Vector3f posFrom, Vector3f posTo, BlockPartFace face, TextureAtlasSprite sprite, EnumFacing facing, ITransformation modelRotationIn, BlockPartRotation partRotation, boolean uvLocked, boolean shade)
     {
