--- /net/minecraft/world/gen/structure/MapGenStructure.java
+++ /net/minecraft/world/gen/structure/MapGenStructure.java
@@ -79,7 +79,7 @@
                 crashreportcategory.addCrashSectionCallable("Structure type", new Callable()
                 {
                     @SideOnly(Side.CLIENT)
-                    final MapGenStructure field_85161_a;
+                    //final MapGenStructure field_85161_a;
                     private static final String __OBFID = "CL_00000508";
                     public String call()
                     {
