--- /net/minecraft/client/renderer/texture/TextureMap.java
+++ /net/minecraft/client/renderer/texture/TextureMap.java
@@ -163,14 +163,14 @@
                 try
                 {
                     IResource iresource = resourceManager.getResource(resourcelocation1);
-                    BufferedImage[] abufferedimage = new BufferedImage[1 + this.mipmapLevels];
-                    abufferedimage[0] = TextureUtil.readBufferedImage(iresource.getInputStream());
+                    BufferedImage[] abufferedimage2 = new BufferedImage[1 + this.mipmapLevels];
+                    abufferedimage2[0] = TextureUtil.readBufferedImage(iresource.getInputStream());
 
                     if (this.mipmapLevels > 0 && abufferedimage != null)
                     {
-                        sheetHeight = abufferedimage[0].getWidth();
-                        abufferedimage[0] = TextureUtils.scaleToPowerOfTwo(abufferedimage[0], minSpriteSize);
-                        int listSprites = abufferedimage[0].getWidth();
+                        sheetHeight = abufferedimage2[0].getWidth();
+                        abufferedimage2[0] = TextureUtils.scaleToPowerOfTwo(abufferedimage2[0], minSpriteSize);
+                        int listSprites = abufferedimage2[0].getWidth();
 
                         if (!TextureUtils.isPowerOfTwo(sheetHeight))
                         {
@@ -187,8 +187,8 @@
 
                         if (!listSprites1.isEmpty())
                         {
-                            int tas = abufferedimage[0].getWidth();
-                            it = abufferedimage[0].getHeight();
+                            int tas = abufferedimage2[0].getWidth();
+                            it = abufferedimage2[0].getHeight();
 
                             if (MathHelper.roundUpToPowerOfTwo(tas) != tas || MathHelper.roundUpToPowerOfTwo(it) != it)
                             {
@@ -202,13 +202,13 @@
                         {
                             it = ((Integer)tas1.next()).intValue();
 
-                            if (it > 0 && it < abufferedimage.length - 1 && abufferedimage[it] == null)
+                            if (it > 0 && it < abufferedimage2.length - 1 && abufferedimage2[it] == null)
                             {
                                 ResourceLocation ss = this.completeResourceLocation(resourcelocation, it);
 
                                 try
                                 {
-                                    abufferedimage[it] = TextureUtil.readBufferedImage(resourceManager.getResource(ss).getInputStream());
+                                    abufferedimage2[it] = TextureUtil.readBufferedImage(resourceManager.getResource(ss).getInputStream());
                                 }
                                 catch (IOException ioexception)
                                 {
@@ -219,7 +219,7 @@
                     }
 
                     AnimationMetadataSection listSprites2 = (AnimationMetadataSection)iresource.getMetadata("animation");
-                    textureatlassprite1.loadSprite(abufferedimage, listSprites2);
+                    textureatlassprite1.loadSprite(abufferedimage2, listSprites2);
                 }
                 catch (RuntimeException runtimeexception)
                 {
