--- /net/minecraft/client/renderer/texture/TextureAtlasSprite.java
+++ /net/minecraft/client/renderer/texture/TextureAtlasSprite.java
@@ -315,17 +315,17 @@
 
         for (k = 0; k < p_180598_1_.length; ++k)
         {
-            BufferedImage i = p_180598_1_[k];
+            BufferedImage i2 = p_180598_1_[k];
 
-            if (i != null)
+            if (i2 != null)
             {
-                if (k > 0 && (i.getWidth() != i >> k || i.getHeight() != j >> k))
+                if (k > 0 && (i2.getWidth() != i >> k || i2.getHeight() != j >> k))
                 {
-                    throw new RuntimeException(String.format("Unable to load miplevel: %d, image is size: %dx%d, expected %dx%d", new Object[] {Integer.valueOf(k), Integer.valueOf(i.getWidth()), Integer.valueOf(i.getHeight()), Integer.valueOf(i >> k), Integer.valueOf(j >> k)}));
+                    throw new RuntimeException(String.format("Unable to load miplevel: %d, image is size: %dx%d, expected %dx%d", new Object[] {Integer.valueOf(k), Integer.valueOf(i2.getWidth()), Integer.valueOf(i2.getHeight()), Integer.valueOf(i >> k), Integer.valueOf(j >> k)}));
                 }
 
-                aint[k] = new int[i.getWidth() * i.getHeight()];
-                i.getRGB(0, 0, i.getWidth(), i.getHeight(), aint[k], 0, i.getWidth());
+                aint[k] = new int[i2.getWidth() * i2.getHeight()];
+                i2.getRGB(0, 0, i2.getWidth(), i2.getHeight(), aint[k], 0, i2.getWidth());
             }
         }
 
