--- /net/minecraft/client/renderer/WorldRenderer.java
+++ /net/minecraft/client/renderer/WorldRenderer.java
@@ -453,8 +453,8 @@
         for (int i = 0; i < listSize; ++i)
         {
             VertexFormatElement vertexformatelement = (VertexFormatElement)list.get(i);
-            int i = vertexformatelement.getOffset() >> 2;
-            int j = this.rawBufferIndex + i;
+            int i2 = vertexformatelement.getOffset() >> 2;
+            int j = this.rawBufferIndex + i2;
 
             switch (WorldRenderer.SwitchEnumUsage.VALUES[vertexformatelement.getUsage().ordinal()])
             {
