--- ../src-base/minecraft/net/minecraft/src/BlockModelUtils.java
+++ ../src-work/minecraft/net/minecraft/src/BlockModelUtils.java
@@ -55,7 +55,7 @@
         boolean flag = false;
         boolean flag1 = true;
         FaceBakery facebakery = new FaceBakery();
-        BakedQuad bakedquad = facebakery.func_178414_a(vector3f, vector3f1, blockpartface, p_makeBakedQuad_1_, p_makeBakedQuad_0_, modelrotation, blockpartrotation, flag, flag1);
+        BakedQuad bakedquad = facebakery.makeBakedQuad(vector3f, vector3f1, blockpartface, p_makeBakedQuad_1_, p_makeBakedQuad_0_, modelrotation, blockpartrotation, flag, flag1);
         return bakedquad;
     }
 }
