--- ../src-base/minecraft/net/minecraft/src/ConnectedProperties.java
+++ ../src-work/minecraft/net/minecraft/src/ConnectedProperties.java
@@ -349,7 +349,7 @@
         }
     }
 
-    public static IProperty getProperty(String p_getProperty_0_, Collection p_getProperty_1_)
+    public static IProperty getProperty(String p_getProperty_0_, Collection<IProperty> p_getProperty_1_)
     {
         for (IProperty iproperty : p_getProperty_1_)
         {
