--- ../src-base/minecraft/net/minecraft/src/ConnectedParser.java
+++ ../src-work/minecraft/net/minecraft/src/ConnectedParser.java
@@ -81,7 +81,7 @@
 
                 if (amatchblock != null)
                 {
-                    list.addAll(Arrays.<E>asList(amatchblock));
+                    list.addAll(Arrays.asList(amatchblock));
                 }
             }
 
@@ -416,7 +416,7 @@
         return comparable;
     }
 
-    public static Comparable getPropertyValue(String p_getPropertyValue_0_, Collection p_getPropertyValue_1_)
+    public static Comparable getPropertyValue(String p_getPropertyValue_0_, Collection<Comparable> p_getPropertyValue_1_)
     {
         for (Comparable comparable : p_getPropertyValue_1_)
         {
