--- ../src-base/minecraft/net/minecraft/src/CustomItemProperties.java
+++ ../src-work/minecraft/net/minecraft/src/CustomItemProperties.java
@@ -340,7 +340,7 @@
         }
         else
         {
-            Set set = map.keySet();
+            Set<String> set = map.keySet();
             Map map1 = new LinkedHashMap();
 
             for (String s1 : set)
@@ -424,7 +424,7 @@
             Set set = map.keySet();
             Map map1 = new LinkedHashMap();
 
-            for (String s1 : set)
+            for (String s1 : (Iterable<String>) set)
             {
                 String s2 = (String)map.get(s1);
                 s2 = fixModelName(s2, p_parseModels_1_);
@@ -622,7 +622,7 @@
     private NbtTagValue[] parseNbtTagValues(Properties p_parseNbtTagValues_1_)
     {
         String s = "nbt.";
-        Map map = getMatchingProperties(p_parseNbtTagValues_1_, s);
+        Map<String, Object> map = getMatchingProperties(p_parseNbtTagValues_1_, s);
 
         if (map.size() <= 0)
         {
@@ -649,7 +649,7 @@
     {
         Map map = new LinkedHashMap();
 
-        for (String s : p_getMatchingProperties_0_.keySet())
+        for (String s : p_getMatchingProperties_0_.stringPropertyNames())
         {
             String s1 = p_getMatchingProperties_0_.getProperty(s);
 
