--- ../src-base/minecraft/net/minecraft/src/ResourceUtils.java
+++ ../src-work/minecraft/net/minecraft/src/ResourceUtils.java
@@ -14,7 +14,7 @@
 public class ResourceUtils
 {
     private static ReflectorClass ForgeAbstractResourcePack = new ReflectorClass(AbstractResourcePack.class);
-    private static ReflectorField ForgeAbstractResourcePack_resourcePackFile = new ReflectorField(ForgeAbstractResourcePack, "field_110597_b");
+    private static ReflectorField ForgeAbstractResourcePack_resourcePackFile = new ReflectorField(ForgeAbstractResourcePack, "resourcePackFile");
     private static boolean directResourcePack = true;
     private static Map localeProperties = null;
 
