--- ../src-base/minecraft/net/minecraft/src/TextureUtils.java
+++ ../src-work/minecraft/net/minecraft/src/TextureUtils.java
@@ -277,9 +277,9 @@
             IReloadableResourceManager ireloadableresourcemanager = (IReloadableResourceManager)iresourcemanager;
             IResourceManagerReloadListener iresourcemanagerreloadlistener = new IResourceManagerReloadListener()
             {
-                public void func_110549_a(IResourceManager p_110549_1_)
+                public void func_110549_a(IResourceManager resourceManager)
                 {
-                    TextureUtils.resourcesReloaded(p_110549_1_);
+                    TextureUtils.resourcesReloaded(resourceManager);
                 }
             };
             ireloadableresourcemanager.func_110542_a(iresourcemanagerreloadlistener);
@@ -291,14 +291,14 @@
             {
                 TextureAnimations.updateCustomAnimations();
             }
-            public void func_110551_a(IResourceManager p_110551_1_) throws IOException
+            public void func_110551_a(IResourceManager resourceManager) throws IOException
             {
             }
             public int func_110552_b()
             {
                 return 0;
             }
-            public void func_174936_b(boolean p_174936_1_, boolean p_174936_2_)
+            public void func_174936_b(boolean blurIn, boolean mipmapIn)
             {
             }
             public void func_174935_a()
