--- ../src-base/minecraft/net/minecraft/world/gen/layer/GenLayer.java
+++ ../src-work/minecraft/net/minecraft/world/gen/layer/GenLayer.java
@@ -49,11 +49,9 @@
 
         GenLayer lvt_7_1_ = GenLayerZoom.func_75915_a(1000L, genlayer4, 0);
         GenLayerRiverInit genlayerriverinit = new GenLayerRiverInit(100L, lvt_7_1_);
-        GenLayerBiome lvt_8_1_ = new GenLayerBiome(200L, genlayer4, p_180781_2_, p_180781_3_);
-        GenLayer genlayer6 = GenLayerZoom.func_75915_a(1000L, lvt_8_1_, 2);
-        GenLayerBiomeEdge genlayerbiomeedge = new GenLayerBiomeEdge(1000L, genlayer6);
+        GenLayer genlayerbiomeedge = p_180781_2_.getBiomeLayer(p_180781_0_, genlayer4, p_180781_3_);
         GenLayer lvt_9_1_ = GenLayerZoom.func_75915_a(1000L, genlayerriverinit, 2);
-        GenLayerHills genlayerhills = new GenLayerHills(1000L, genlayerbiomeedge, lvt_9_1_);
+        GenLayer genlayerhills = new GenLayerHills(1000L, genlayerbiomeedge, lvt_9_1_);
         GenLayer genlayer5 = GenLayerZoom.func_75915_a(1000L, genlayerriverinit, 2);
         genlayer5 = GenLayerZoom.func_75915_a(1000L, genlayer5, j);
         GenLayerRiver genlayerriver = new GenLayerRiver(1L, genlayer5);
