--- ../src-base/minecraft/net/minecraft/client/gui/FontRenderer.java
+++ ../src-work/minecraft/net/minecraft/client/gui/FontRenderer.java
@@ -757,7 +757,7 @@
         this.field_78294_m = p_78275_1_;
     }
 
-    public List func_78271_c(String p_78271_1_, int p_78271_2_)
+    public List<String> func_78271_c(String p_78271_1_, int p_78271_2_)
     {
         return Arrays.asList(this.func_78280_d(p_78271_1_, p_78271_2_).split("\n"));
     }
@@ -949,8 +949,9 @@
                 Properties properties = new Properties();
                 properties.load(inputstream);
 
-                for (String s3 : properties.keySet())
+                for (Object a : properties.keySet())
                 {
+                	String s3 = (String) a;
                     String s4 = "width.";
 
                     if (s3.startsWith(s4))
