--- ../src-base/minecraft/net/minecraft/client/renderer/block/model/FaceBakery.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/block/model/FaceBakery.java
@@ -24,28 +24,28 @@
     private static final float field_178418_a = 1.0F / (float)Math.cos(0.39269909262657166D) - 1.0F;
     private static final float field_178417_b = 1.0F / (float)Math.cos((Math.PI / 4D)) - 1.0F;
     private static final FaceBakery.Rotation[] field_188016_c = new FaceBakery.Rotation[ModelRotation.values().length * EnumFacing.values().length];
-    private static final FaceBakery.Rotation field_188017_d = new FaceBakery.Rotation(null)
+    private static final FaceBakery.Rotation field_188017_d = new FaceBakery.Rotation()
     {
         BlockFaceUV func_188007_a(float p_188007_1_, float p_188007_2_, float p_188007_3_, float p_188007_4_)
         {
             return new BlockFaceUV(new float[] {p_188007_1_, p_188007_2_, p_188007_3_, p_188007_4_}, 0);
         }
     };
-    private static final FaceBakery.Rotation field_188018_e = new FaceBakery.Rotation(null)
+    private static final FaceBakery.Rotation field_188018_e = new FaceBakery.Rotation()
     {
         BlockFaceUV func_188007_a(float p_188007_1_, float p_188007_2_, float p_188007_3_, float p_188007_4_)
         {
             return new BlockFaceUV(new float[] {p_188007_4_, 16.0F - p_188007_1_, p_188007_2_, 16.0F - p_188007_3_}, 270);
         }
     };
-    private static final FaceBakery.Rotation field_188019_f = new FaceBakery.Rotation(null)
+    private static final FaceBakery.Rotation field_188019_f = new FaceBakery.Rotation()
     {
         BlockFaceUV func_188007_a(float p_188007_1_, float p_188007_2_, float p_188007_3_, float p_188007_4_)
         {
             return new BlockFaceUV(new float[] {16.0F - p_188007_1_, 16.0F - p_188007_2_, 16.0F - p_188007_3_, 16.0F - p_188007_4_}, 0);
         }
     };
-    private static final FaceBakery.Rotation field_188020_g = new FaceBakery.Rotation(null)
+    private static final FaceBakery.Rotation field_188020_g = new FaceBakery.Rotation()
     {
         BlockFaceUV func_188007_a(float p_188007_1_, float p_188007_2_, float p_188007_3_, float p_188007_4_)
         {
@@ -55,30 +55,30 @@
 
     public BakedQuad func_178414_a(Vector3f p_178414_1_, Vector3f p_178414_2_, BlockPartFace p_178414_3_, TextureAtlasSprite p_178414_4_, EnumFacing p_178414_5_, ModelRotation p_178414_6_, @Nullable BlockPartRotation p_178414_7_, boolean p_178414_8_, boolean p_178414_9_)
     {
-        return this.makeBakedQuad(p_178414_1_, p_178414_2_, p_178414_3_, p_178414_4_, p_178414_5_, p_178414_6_, p_178414_7_, p_178414_8_, p_178414_9_);
+        return this.makeBakedQuad(p_178414_1_, p_178414_2_, p_178414_3_, p_178414_4_, p_178414_5_, (ITransformation) p_178414_6_, p_178414_7_, p_178414_8_, p_178414_9_);
     }
 
-    public BakedQuad makeBakedQuad(Vector3f p_makeBakedQuad_1_, Vector3f p_makeBakedQuad_2_, BlockPartFace p_makeBakedQuad_3_, TextureAtlasSprite p_makeBakedQuad_4_, EnumFacing p_makeBakedQuad_5_, ITransformation p_makeBakedQuad_6_, BlockPartRotation p_makeBakedQuad_7_, boolean p_makeBakedQuad_8_, boolean p_makeBakedQuad_9_)
+    public BakedQuad makeBakedQuad(Vector3f p_178414_1_, Vector3f p_178414_2_, BlockPartFace p_178414_3_, TextureAtlasSprite p_178414_4_, EnumFacing p_178414_5_, ITransformation p_178414_6_, BlockPartRotation p_178414_7_, boolean p_178414_8_, boolean p_178414_9_)
     {
-        BlockFaceUV blockfaceuv = p_makeBakedQuad_3_.field_178243_e;
+        BlockFaceUV blockfaceuv = p_178414_3_.field_178243_e;
 
-        if (p_makeBakedQuad_8_)
+        if (p_178414_8_)
         {
             if (Reflector.ForgeHooksClient_applyUVLock.exists())
             {
-                blockfaceuv = (BlockFaceUV)Reflector.call(Reflector.ForgeHooksClient_applyUVLock, new Object[] {p_makeBakedQuad_3_.field_178243_e, p_makeBakedQuad_5_, p_makeBakedQuad_6_});
+                blockfaceuv = (BlockFaceUV)Reflector.call(Reflector.ForgeHooksClient_applyUVLock, new Object[] {p_178414_3_.field_178243_e, p_178414_5_, p_178414_6_});
             }
             else
             {
-                blockfaceuv = this.func_188010_a(p_makeBakedQuad_3_.field_178243_e, p_makeBakedQuad_5_, (ModelRotation)p_makeBakedQuad_6_);
+                blockfaceuv = this.func_188010_a(p_178414_3_.field_178243_e, p_178414_5_, (ModelRotation)p_178414_6_);
             }
         }
 
-        boolean flag = p_makeBakedQuad_9_ && !Reflector.ForgeHooksClient_fillNormal.exists();
-        int[] aint = this.makeQuadVertexData(blockfaceuv, p_makeBakedQuad_4_, p_makeBakedQuad_5_, this.func_178403_a(p_makeBakedQuad_1_, p_makeBakedQuad_2_), p_makeBakedQuad_6_, p_makeBakedQuad_7_, flag);
+        boolean flag = p_178414_9_ && !Reflector.ForgeHooksClient_fillNormal.exists();
+        int[] aint = this.makeQuadVertexData(blockfaceuv, p_178414_4_, p_178414_5_, this.func_178403_a(p_178414_1_, p_178414_2_), p_178414_6_, p_178414_7_, flag);
         EnumFacing enumfacing = func_178410_a(aint);
 
-        if (p_makeBakedQuad_7_ == null)
+        if (p_178414_7_ == null)
         {
             this.func_178408_a(aint, enumfacing);
         }
@@ -86,11 +86,11 @@
         if (Reflector.ForgeHooksClient_fillNormal.exists())
         {
             Reflector.call(Reflector.ForgeHooksClient_fillNormal, new Object[] {aint, enumfacing});
-            return new BakedQuad(aint, p_makeBakedQuad_3_.field_178245_c, enumfacing, p_makeBakedQuad_4_, p_makeBakedQuad_9_, DefaultVertexFormats.field_176599_b);
+            return new BakedQuad(aint, p_178414_3_.field_178245_c, enumfacing, p_178414_4_, p_178414_9_, DefaultVertexFormats.field_176599_b);
         }
         else
         {
-            return new BakedQuad(aint, p_makeBakedQuad_3_.field_178245_c, enumfacing, p_makeBakedQuad_4_);
+            return new BakedQuad(aint, p_178414_3_.field_178245_c, enumfacing, p_178414_4_);
         }
     }
 
@@ -125,9 +125,9 @@
         return -16777216 | i << 16 | i << 8 | i;
     }
 
-    public static float func_178412_b(EnumFacing p_178412_0_)
+    public static float func_178412_b(EnumFacing p_178412_1_)
     {
-        switch (p_178412_0_)
+        switch (p_178412_1_)
         {
             case DOWN:
 
@@ -174,16 +174,16 @@
         return afloat;
     }
 
-    private void fillVertexData(int[] p_fillVertexData_1_, int p_fillVertexData_2_, EnumFacing p_fillVertexData_3_, BlockFaceUV p_fillVertexData_4_, float[] p_fillVertexData_5_, TextureAtlasSprite p_fillVertexData_6_, ITransformation p_fillVertexData_7_, @Nullable BlockPartRotation p_fillVertexData_8_, boolean p_fillVertexData_9_)
+    private void fillVertexData(int[] p_188015_1_, int p_188015_2_, EnumFacing p_188015_3_, BlockFaceUV p_188015_4_, float[] p_188015_5_, TextureAtlasSprite p_188015_6_, ITransformation p_188015_7_, @Nullable BlockPartRotation p_188015_8_, boolean p_188015_9_)
     {
-        EnumFacing enumfacing = p_fillVertexData_7_.rotate(p_fillVertexData_3_);
-        int i = p_fillVertexData_9_ ? this.func_178413_a(enumfacing) : -1;
-        EnumFaceDirection.VertexInformation enumfacedirection$vertexinformation = EnumFaceDirection.func_179027_a(p_fillVertexData_3_).func_179025_a(p_fillVertexData_2_);
-        Vector3f vector3f = new Vector3f(p_fillVertexData_5_[enumfacedirection$vertexinformation.field_179184_a], p_fillVertexData_5_[enumfacedirection$vertexinformation.field_179182_b], p_fillVertexData_5_[enumfacedirection$vertexinformation.field_179183_c]);
-        this.func_178407_a(vector3f, p_fillVertexData_8_);
-        int j = this.rotateVertex(vector3f, p_fillVertexData_3_, p_fillVertexData_2_, p_fillVertexData_7_);
+        EnumFacing enumfacing = p_188015_7_.rotate(p_188015_3_);
+        int i = p_188015_9_ ? this.func_178413_a(enumfacing) : -1;
+        EnumFaceDirection.VertexInformation enumfacedirection$vertexinformation = EnumFaceDirection.func_179027_a(p_188015_3_).func_179025_a(p_188015_2_);
+        Vector3f vector3f = new Vector3f(p_188015_5_[enumfacedirection$vertexinformation.field_179184_a], p_188015_5_[enumfacedirection$vertexinformation.field_179182_b], p_188015_5_[enumfacedirection$vertexinformation.field_179183_c]);
+        this.func_178407_a(vector3f, p_188015_8_);
+        int j = this.rotateVertex(vector3f, p_188015_3_, p_188015_2_, p_188015_7_);
         BlockModelUtils.snapVertexPosition(vector3f);
-        this.func_178404_a(p_fillVertexData_1_, j, p_fillVertexData_2_, vector3f, i, p_fillVertexData_6_, p_fillVertexData_4_);
+        this.func_178404_a(p_188015_1_, j, p_188015_2_, vector3f, i, p_188015_6_, p_188015_4_);
     }
 
     private void func_178404_a(int[] p_178404_1_, int p_178404_2_, int p_178404_3_, Vector3f p_178404_4_, int p_178404_5_, TextureAtlasSprite p_178404_6_, BlockFaceUV p_178404_7_)
@@ -244,7 +244,7 @@
 
     public int func_188011_a(Vector3f p_188011_1_, EnumFacing p_188011_2_, int p_188011_3_, ModelRotation p_188011_4_)
     {
-        return this.rotateVertex(p_188011_1_, p_188011_2_, p_188011_3_, p_188011_4_);
+        return this.func_188011_a(p_188011_1_, p_188011_2_, p_188011_3_, p_188011_4_);
     }
 
     public int rotateVertex(Vector3f p_rotateVertex_1_, EnumFacing p_rotateVertex_2_, int p_rotateVertex_3_, ITransformation p_rotateVertex_4_)
