--- ../src-base/minecraft/net/minecraft/client/renderer/block/model/ModelRotation.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/block/model/ModelRotation.java
@@ -122,7 +122,7 @@
 
     public javax.vecmath.Matrix4f getMatrix()
     {
-        return Reflector.ForgeHooksClient_getMatrix.exists() ? (javax.vecmath.Matrix4f)Reflector.call(Reflector.ForgeHooksClient_getMatrix, new Object[] {this}): new javax.vecmath.Matrix4f(this.func_177525_a());
+        return Reflector.ForgeHooksClient_getMatrix.exists() ? (javax.vecmath.Matrix4f)Reflector.call(Reflector.ForgeHooksClient_getMatrix, new Object[] {this}): cloneMatrix(this.func_177525_a());
     }
 
     public EnumFacing rotate(EnumFacing p_rotate_1_)
@@ -142,4 +142,29 @@
             field_177546_q.put(Integer.valueOf(modelrotation.field_177545_r), modelrotation);
         }
     }
+    
+    public static javax.vecmath.Matrix4f cloneMatrix(Matrix4f m1)
+    {
+    	javax.vecmath.Matrix4f ret = new javax.vecmath.Matrix4f();
+        ret.m00 = m1.m00;
+        ret.m01 = m1.m01;
+        ret.m02 = m1.m02;
+        ret.m03 = m1.m03;
+
+        ret.m10 = m1.m10;
+        ret.m11 = m1.m11;
+        ret.m12 = m1.m12;
+        ret.m13 = m1.m13;
+
+        ret.m20 = m1.m20;
+        ret.m21 = m1.m21;
+        ret.m22 = m1.m22;
+        ret.m23 = m1.m23;
+
+        ret.m30 = m1.m30;
+        ret.m31 = m1.m31;
+        ret.m32 = m1.m32;
+        ret.m33 = m1.m33;
+        return ret;
+    }
 }
