--- ../src-base/minecraft/net/minecraft/profiler/Profiler.java
+++ ../src-work/minecraft/net/minecraft/profiler/Profiler.java
@@ -22,7 +22,7 @@
     private final List field_76326_c = Lists.newArrayList();
     public boolean field_76327_a;
     private String field_76323_d = "";
-    private final Map field_76324_e = Maps.newHashMap();
+    private final Map<String, Object> field_76324_e = Maps.newHashMap();
     @SideOnly(Side.CLIENT)
     public boolean profilerGlobalEnabled = true;
     @SideOnly(Side.CLIENT)
