--- ../src-base/minecraft/net/minecraft/world/World.java
+++ ../src-work/minecraft/net/minecraft/world/World.java
@@ -1824,7 +1824,7 @@
                     {
                         this.field_72984_F.func_194340_a(() ->
                         {
-                            return String.valueOf((Object)TileEntity.func_190559_a(lvt_2_4_.getClass()));
+                            return String.valueOf((Object)TileEntity.func_190559_a(tileentity.getClass()));
                         });
                         ((ITickable)tileentity).func_73660_a();
                         this.field_72984_F.func_76319_b();
@@ -3014,9 +3014,9 @@
 
         for (Entity entity4 : this.field_72996_f)
         {
-            if (p_175644_1_.isAssignableFrom(entity4.getClass()) && p_175644_2_.apply(entity4))
+            if (p_175644_1_.isAssignableFrom(entity4.getClass()) && p_175644_2_.apply((T) entity4))
             {
-                list.add(entity4);
+                list.add((T) entity4);
             }
         }
 
@@ -3029,9 +3029,9 @@
 
         for (Entity entity4 : this.field_73010_i)
         {
-            if (p_175661_1_.isAssignableFrom(entity4.getClass()) && p_175661_2_.apply(entity4))
+            if (p_175661_1_.isAssignableFrom(entity4.getClass()) && p_175661_2_.apply((T) entity4))
             {
-                list.add(entity4);
+                list.add((T) entity4);
             }
         }
 
@@ -3074,7 +3074,7 @@
 
         for (int j2 = 0; j2 < list.size(); ++j2)
         {
-            T t1 = (Entity)list.get(j2);
+            T t1 = (T)list.get(j2);
 
             if (t1 != p_72857_3_ && EntitySelectors.field_180132_d.apply(t1))
             {
@@ -3147,7 +3147,7 @@
     {
         IBlockState iblockstate1 = this.func_180495_p(p_190527_2_);
         AxisAlignedBB axisalignedbb = p_190527_3_ ? null : p_190527_1_.func_176223_P().func_185890_d(this, p_190527_2_);
-        return axisalignedbb != Block.field_185506_k && !this.func_72917_a(axisalignedbb.func_186670_a(p_190527_2_), p_190527_5_) ? false : (iblockstate1.func_185904_a() == Material.field_151594_q && p_190527_1_ == Blocks.field_150467_bQ ? true : iblockstate1.func_185904_a().func_76222_j() && p_190527_1_.func_176198_a(this, p_190527_2_, p_190527_4_));
+        return axisalignedbb != Block.field_185506_k && !this.func_72917_a(axisalignedbb.func_186670_a(p_190527_2_), p_190527_5_) ? false : (iblockstate1.func_185904_a() == Material.field_151594_q && p_190527_1_ == Blocks.field_150467_bQ ? true : iblockstate1.func_177230_c().func_176200_f(this, p_190527_2_) && p_190527_1_.func_176198_a(this, p_190527_2_, p_190527_4_));
     }
 
     public int func_181545_F()
