--- ../src-base/minecraft/shadersmod/client/Shaders.java
+++ ../src-work/minecraft/shadersmod/client/Shaders.java
@@ -58,6 +58,7 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 import org.lwjgl.BufferUtils;
+import org.lwjgl.opengl.ARBFragmentShader;
 import org.lwjgl.opengl.ARBShaderObjects;
 import org.lwjgl.opengl.ARBVertexShader;
 import org.lwjgl.opengl.ContextCapabilities;
@@ -3090,7 +3091,7 @@
     {
         if (currentWorld != world)
         {
-            World world = currentWorld;
+            World world2 = currentWorld;
             currentWorld = world;
 
             if (world != null && world != null)
@@ -4232,7 +4233,7 @@
                         Lang.loadLocaleData(inputstream, properties);
                         inputstream.close();
 
-                        for (String s4 : properties.keySet())
+                        for (String s4 : properties.stringPropertyNames())
                         {
                             String s5 = properties.getProperty(s4);
                             shaderPackResources.put(s4, s5);
