--- ../src-base/minecraft/net/minecraft/client/renderer/texture/TextureMap.java
+++ ../src-work/minecraft/net/minecraft/client/renderer/texture/TextureMap.java
@@ -83,7 +83,7 @@
         this(p_i46100_1_, p_i46100_2_, false);
     }
 
-    public TextureMap(String p_i1_1_, ITextureMapPopulator p_i1_2_, boolean p_i1_3_)
+    public TextureMap(String p_i46100_1_, ITextureMapPopulator p_i46100_2_, boolean skipFirst)
     {
         this.iconGrid = null;
         this.iconGridSize = -1;
@@ -98,8 +98,8 @@
         this.field_110574_e = Maps.<String, TextureAtlasSprite>newHashMap();
         this.field_94252_e = Maps.<String, TextureAtlasSprite>newHashMap();
         this.field_94249_f = new TextureAtlasSprite("missingno");
-        this.field_94254_c = p_i1_1_;
-        this.field_174946_m = p_i1_2_;
+        this.field_94254_c = p_i46100_1_;
+        this.field_174946_m = p_i46100_2_;
     }
 
     private void func_110569_e()
@@ -743,7 +743,7 @@
         return l3;
     }
 
-    private int detectMinimumSpriteSize(Map p_detectMinimumSpriteSize_1_, IResourceManager p_detectMinimumSpriteSize_2_, int p_detectMinimumSpriteSize_3_)
+    private int detectMinimumSpriteSize(Map<Object, Object> p_detectMinimumSpriteSize_1_, IResourceManager p_detectMinimumSpriteSize_2_, int p_detectMinimumSpriteSize_3_)
     {
         Map map1 = new HashMap();
 
