--- ../src-base/minecraft/net/minecraft/client/particle/EffectRenderer.java
+++ ../src-work/minecraft/net/minecraft/client/particle/EffectRenderer.java
@@ -39,7 +39,7 @@
     private static final ResourceLocation field_110737_b = new ResourceLocation("textures/particle/particles.png");
     protected World field_78878_a;
     private List[][] field_78876_b = new List[4][];
-    private List field_178933_d = Lists.newArrayList();
+    private List<EntityParticleEmitter> field_178933_d = Lists.newArrayList();
     private TextureManager field_78877_c;
     private Random field_78875_d = new Random();
     private Map field_178932_g = Maps.newHashMap();
