--- ../src-base/minecraft/net/minecraft/potion/PotionHelper.java
+++ ../src-work/minecraft/net/minecraft/potion/PotionHelper.java
@@ -53,7 +53,7 @@
         return func_77908_a(p_77909_0_, 5, 4, 3, 2, 1);
     }
 
-    public static int func_77911_a(Collection p_77911_0_)
+    public static int func_77911_a(Collection<PotionEffect> p_77911_0_)
     {
         int i = 3694022;
 
@@ -108,7 +108,7 @@
         }
     }
 
-    public static boolean func_82817_b(Collection p_82817_0_)
+    public static boolean func_82817_b(Collection<PotionEffect> p_82817_0_)
     {
         for (PotionEffect potioneffect : p_82817_0_)
         {
